### PR TITLE
feat: redirect llamacpp log

### LINKED ISSUE #1054: improve: redirect log to right position
Currently: 
- all log from cli and cortex.cpp go to log file
- log from engine go to terminal

Target:
- Log from CLI go to terminal
- Log from cortex.cpp server and engines go to log file

### PR DESCRIPTION
**Pull request by nguyenhoangthuan99**
Fix #1054 
related PR https://github.com/janhq/cortex.llamacpp/pull/223
Sumarry changes:
- Call SetFileLogger for cortex.llamacpp engine when engine is loaded.
- Related pr in cortex.llamacpp repo : https://github.com/janhq/cortex.llamacpp/pull/223

### CODE REVIEW

**Comment by namchuai** [comment]
Please squash the commit

File: engine/controllers/server.cc [hunk 1]
```diff
@@ -352,6 +352,11 @@ void server::LoadModel(const HttpRequestPtr& req,
 
   LOG_TRACE << "Load model";
   auto& en = std::get<EngineI*>(engines_[engine_type].engine);
+  if (engine_type == kLlamaEngine) {  //fix for llamacpp engine first
+    auto config = file_manager_utils::GetCortexConfig();
+    en->SetFileLogger(config.maxLogLines, config.logFolderPath + "/" +
```
Comment (namchuai) [inline engine/controllers/server.cc]: nit, IMO using `std::filesystem::path` is safer compare to string concatenation

Comment (nguyenhoangthuan99) [reply engine/controllers/server.cc]: I fixed it in this pr #1200

**Review by dan-menlo — APPROVED** [review]
@nguyenhoangthuan99 Approved to unblock. Please also link the `cortex.llamacpp` PR in this, as I think they have to be read together for future users to understand.

**Review by vansangpfiev — APPROVED** [review]

**Pull request merged**